query_table_create.go

```diff
@@ -309,7 +303,8 @@ func (q *CreateTableQuery) appendFKConstraints(
 }
 
 func (q *CreateTableQuery) appendPKConstraint(b []byte, pks []*schema.Field) []byte {
-	if len(pks) == 0 {
+	// Primary key constraint might've already been created together with AUTOINCREMENT directives.
+	if len(pks) == 0 || bytes.Contains(b, []byte("PRIMARY KEY")) {
```

Comment:
Please move this to the caller side so this method is called only if `...`.

Perhaps we could also make it more clear `bytes.Contains(b, []byte("PRIMARY KEY"))` applies to SQLite.

Reply:
Pushed the check to the caller and added more info in the comment